File: FRONT/src/app/interfaces/news.ts
import { Topics } from "./Topics";

export interface news {
    id: number;
    title: string;
    introduction: string;
    author: string;
    description: string;
    content: string;
    createdAt: string;    
    imageUrl: string;
    topics: Topics[]; 
    likes?: number;
}

File: API/src/Backend/AMS_News.Domain/Entities/News.cs
using System.Diagnostics;

namespace AMS_News.Domain.Entities;

public class News : BaseEntity
{
    public string Title { get; set; } = string.Empty;
    public string Introduction { get; set; } = string.Empty;
    public string Content { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public string Author { get; set; } = string.Empty;
    public IList<Topics> Topics { get; set; } = [];
    public string? ImageIdentifier { get; set; } = string.Empty;
    public long CustomerId { get; set; }
    
}